data_diff/databases/connect.py

```diff
@@ -132,6 +137,12 @@ def connect_to_uri(db_uri: str, thread_count: Optional[int] = 1) -> Database:
             kw["password"] = dsn.password
     kw = {k: v for k, v in kw.items() if v is not None}
 
+    if scheme == "databricks":
+        # dsn user - access token
+        # sdn password - http path (starting with /)
+
+        return cls(http_path=dsn.password, access_token=dsn.user, server_hostname=dsn.hostname, **kw)
```

Comment:
I don't know how I feel about putting the `http_path` as the password.

Can you tell me why you chose this, instead of adding it to the path, or as a query argument?

Reply:
> adding it to the path

http_path starts with `/` symbol so adding it to the path leads to an error when parsing. We can just notify users no to use the slash in their path but I bet what the first run will be always failed because from Databricks UI http_path is copied with the slash so people might be confused during their first attempt of diffing.

> a query argument

I assumed that query params in terms of usage look like additional i.e. not necessarily so people might skip it for the first run, and it'll lead to an error as well.

So, my concern it just not to disappoint people in their first attempt of diffing.

Reply:
We can provide a good error, telling users that this parameter is necessary.

I think it's less confusing than using the password. Also, we're going to remove the password whenever we print the configuration to the logs. That means the http_path will appear as `****`.

P.S. we can also ensure that it's formatted with only one leading slash. shouldn't be too hard to do.

Reply:
Ok, I'll update the uri with a format `databricks://:access_token@account/http_path/catalog/schema/` in that case.

Reply:
Maybe account should be where user is? Seems to make sense.

(I don't know databricks, do they have some convention about this?)